docs/plugins/program_output.js

```diff
@@ -20,9 +32,12 @@ const { readFile, writeFile } = fs.promises;
 
 const PROGRAM_OUTPUT_RE = /```[a-z]+ \[([^\]]+)\]\n```/;
 
+const VERSIONED_DOCS_PATH_RE = /(\/versioned_docs\/version-\d+\.\d+\.x)\//;
```

Comment:
Noticing that when you use these Regexes later in the code you don't need to use the `new RegExp()` constructor again since they're already constructed, you could just add the `gi` tag here, like `PROGRAM_OUTPUT_RE = /```[a-z]+ \[([^\]]+)\]\n```/gi`